pkg/jsonSchemaValidator/validator_test.go

```diff
@@ -49,3 +56,76 @@ func TestValidateCustomKeysPass(t *testing.T) {
 
 	assert.Empty(t, errorsResult)
 }
+
+//go:embed test_fixtures/invalid-rego-definition.json
+var invalidRegoDefinitionJson string
+
+//go:embed test_fixtures/valid-rego-definition.json
+var validRegoDefinitionJson string
+
+//go:embed test_fixtures/rego-rule-fail.yaml
+var regoRuleFail string
+
+//go:embed test_fixtures/rego-rule-pass.yaml
+var regoRulePass string
+
+func TestRegoDefinitionCustomKey(t *testing.T) {
```

Comment:
I have an idea for improving those tests.
The public interface of this package, used by the rest of the code is as follows:
```
validationResult, err := e.jsonSchemaValidator.ValidateYamlSchema(string(ruleSchemaJson), string(configurationJson))
```
in essence: pass a rule's yaml schema, pass a file to test - get a result.

so it makes more sense to test the software the same way the users of the package use it (this time, the users are the pieces of code using this package)